guide/java/archetype.md

```diff
@@ -73,22 +75,31 @@ For example:
 {% highlight yaml %}
 brooklyn.catalog:
     brooklyn.libraries:
-    - file:///path/to/jar/autobrick-0.1.0-SNAPSHOT.jar     
+    - file:///path/to/jar/autobrick-0.1.0-SNAPSHOT.jar
+    version: "0.1.0-SNAPSHOT"
     itemType: entity
     items:
-    - id: com.acme.MySample
+    - id: com.acme.autobrick.MySample
       item:
-        type: com.acme.MySample
+        type: com.acme.autobrick.MySample
```

Comment:
For the next release of Brooklyn where Karaf is the preferred launcher we need to prefix this with the bundle symbolic name.